src/main/java/org/bytedeco/javacv/Frame.java

```diff
@@ -77,10 +77,15 @@
 
     /** Constants defining data type in the frame*/
     public static enum Type {
+    	UNKNOWN,
```

Comment:
Let's make UNKNOWN = -1, making sure VIDEO = 0, and let's also follow the order of FFmpeg, unless there's a good reason not to: AVMEDIA_TYPE_UNKNOWN = -1, AVMEDIA_TYPE_VIDEO, AVMEDIA_TYPE_AUDIO, AVMEDIA_TYPE_DATA, AVMEDIA_TYPE_SUBTITLE, AVMEDIA_TYPE_ATTACHMENT

Reply:
All right. I swapped data and subtitle by accident.

Reply:
And I forgot, but we can't set the ordinal() for enum, so for backward compatibility, we shouldn't put UNKNOWN first. Do we even need an UNKNOWN though? It might be problematic for forward compatibility. We could just initialize the field to null like all the rest of the fields. What do you think?